cli/src/commands/sandbox.rs

```diff
@@ -44,8 +48,18 @@ pub async fn handle_sandbox(matches: &ArgMatches) -> CommandResult {
 /// Lock down the current process.
 #[cfg(unix)]
 fn lock_process(matches: &ArgMatches) -> Result<()> {
-    let mut birdcage =
-        if matches.get_flag("strict") { Birdcage::new()? } else { permissions::default_sandbox()? };
+    let birdcage =
+        if matches.get_flag("strict") { Birdcage::new() } else { permissions::default_sandbox() };
+
+    // Provide additional error context.
+    let mut birdcage = match birdcage {
+        Ok(birdcage) => birdcage,
+        #[cfg(target_os = "linux")]
+        Err(err @ SandboxError::Ruleset(_)) => {
+            return Err(Error::from(err)).context("sandbox requires Linux 5.13+");
```

Comment:
The `return` gets a lot shorter if you capture the entire `Result`

```suggestion
        err @ Err(SandboxError::Ruleset(_)) => {
            return err.context("sandbox requires Linux 5.13+");
```

Reply:
Your suggestion doesn't work since the error isn't the same as the return type?

Reply:
Ah. Good point... Didn't test it because I'm not on Linux...

I guess that would have to be

```
        err @ Err(SandboxError::Ruleset(_)) => err.context("sandbox requires Linux 5.13+")?,
```

But that feels weird....